docs/pages/versions/unversioned/sdk/mesh-gradient.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: MeshGradient
+description: A module that exposes MeshGradient view from SwiftUI to React Native.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-video'
+packageName: 'expo-mesh-gradient'
+iconUrl: '/static/images/packages/expo-mesh-gradient.png'
+platforms: ['ios', 'tvos']
```

Comment:
```suggestion
platforms: ['ios', 'tvos', 'macos']
```

Reply:
Hah, I know that macOS is listed as supported in the podspec, but I'd like to actually test it there before listing it in the docs 😅